mapping/summary_numbers.R

```diff
@@ -0,0 +1,275 @@
+
+# libs --------------------------------------------------------------------
+library(ggplot2)
+library(tidyverse)
+library(sf)
```

Comment:
I am interpreting these to be scratch files. Should they be committed? More information is needed here of somewhere to understand how `out_shp` and `mapping` relate to the pipeline. 

I'd suggest making `out_shop` a subdirectory of `4_export/out` to keep with the pipeline structure, and making output files in a predictable place. The `4_export` directory itself is a bit confusing since typical pipeline structure includes an `out` folder in the different phases, so having it as a phase itself is new to me. I think this makes sense if this is where we want to focus all the exported objects that are intended to be shared back with the project team, so we wouldn't need to search around and can just copy/paste from a single place.